spec/yaml/swiftnav/sbp/ssr.yaml

```
@@ -802,3 +802,120 @@ definitions:
               The spec describes the encoding scheme in detail, but
               essentially the index of the quadrants that contain transitions between
               valid and invalid (and vice versa) are encoded as u8 integers.
+
+ - BoundsHeader:
+     short_desc: Header for the Bounds messages
+     fields:
+       - time:
+           type: GPSTimeSec
+           desc: GNSS reference time of the bound
+       - num_msgs:
```

For consistency, in SBP the prefix `n_` is typically used for these sorts of fields. Such as `n_obs` (observation_header) or `n_group_msgs` (system_meta)

```suggestion
       - n_msgs:
```

---

In the other SSR messages (STEC Correction, Gridded Correction) the field is called `num_msgs` though.